puma/hlplots/results.py

```diff
@@ -0,0 +1,281 @@
+# import h5py
+
+from puma import Histogram, HistogramPlot, Roc, RocPlot, VarVsEff, VarVsEffPlot
+from puma.utils import global_config, logger
+
+
+class Results:
+    """Stores all results of the different taggers."""
+
+    def __init__(self) -> None:
+        self.taggers = []
+        self.model_names = []
+        # defining target efficiency
+        self.sig_eff = None
+        self.atlas_second_tag = None
+        self.plot_name = None
+        self.n_jets_light = None
+        self.n_jets_c = None
+        self.is_light = None
+        self.is_c = None
+        self.is_b = None
```

Comment:
These would also have to be per model if we allowed the results to be loaded from differnet filepaths